src/test/java/com/example/MateApplicationTests.java

```diff
@@ -0,0 +1,12 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class MateApplicationTests {
```

Comment:
Don't use `Mate` in this project, it will go to your portfolio. Rename this repo too. Also rename com.example to smth more relevant